templates/backup-pre-postgres.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+TMPDIR="{{ POSTGRES_SERVER_BACKUP_DIR }}"
+
+rm -rf "$TMPDIR"
+mkdir -p "$TMPDIR"
+sudo -u postgres pg_dumpall > "$TMPDIR/postgresql_all_dbs.sql"
```

Comment:
We might consider splitting the backups to match ``mysql`` role (restoring single database might be easier then). It's not really a requirement, rather just point that two database roles do things differently here.